vs.target.js

```diff
@@ -140,6 +140,41 @@
 
     const onresult = function (event) {
         var keyword = event.results[0][0].transcript.toLowerCase();
+        var arr = [
+        "\\bsearch\\b", "\\bfor\\b", "\\bshow\\b", "\\bfind\\b", "\\bget\\b", "\\bextract\\b", "\\bgive\\b" , "\\bsame\\b", "\\bsimilar\\b", "\\bwant\\b", "\\bwants\\b", "\\bme\\b", "\\bi\\b", "\\bfetch\\b", "\\bdo\\b", "\\btake\\b", "\\bto\\b", "\\bgo\\b", "\\bsome\\b"
+        ], arrRegex = new RegExp(arr.join('|'), 'g');
+        keyword = keyword.replace(arrRegex, '');
```

Comment:
const command = keyword.match(/cart|account|wishlist|sign in|sign up|track my order|track order|help|order/g);

if(command){


}